### DISCUSSION TITLE
.NET 5.0 application deploy by ClickOnce: how to create deployment web page (publish.htm)?

### DISCUSSION THREAD

**Discussion by worldofsites** [dotnet/aspnetcore#27761] [Q&A] [answered]
In previous versions of ClickOnce deploy we can configure deployment web page:
![image](https://user-images.githubusercontent.com/23352829/98891846-a2060380-24d1-11eb-8172-1f17f5500e46.png)
But in new version we can't:
![image](https://user-images.githubusercontent.com/23352829/98891870-ac280200-24d1-11eb-9cf8-3e240589e437.png)
Is there any another way to create deployment web page?

**Comment by Pilchie** [comment, marked as answer]
@John-Hart - any idea where this question should be asked?

**Reply by John-Hart** [reply]
The best place for these questions is https://developercommunity.visualstudio.com

We'll look at adding this feature in a future update.  For now, I've created an issue for this with a simple work around: 

https://developercommunity.visualstudio.com/content/problem/1254600/the-clickonce-publish-wizard-for-net-core-31-and-n-1.html.

**Reply by Pilchie** [reply]
Thanks - I'm going to mark that as the Answer

**Reply by worldofsites** [reply]
Thanks @John-Hart @Pilchie
